docs/docs/30-administration/11-forges/30-gitea.md

```diff
@@ -20,6 +20,24 @@ services:
     [...]
 ```
 
+## Gitea on the same host
+
+If you have Gitea also running on the same host within a container, make sure the agent does have access to it.
+The agent try to clone by the clone URL Gitea reports via API. So you might want to specify a docker network for the agent too:
+
+```diff
+# docker-compose.yml
+version: '3'
+
+services:
+  [...]
+  woodpecker-agent:
+    [...]
+    environment:
+      - [...]
++     - WOODPECKER_BACKEND_DOCKER_NETWORK=gitea
```

Comment:
Needs to be `woodpecker` or we should adjust the sample compose file to use `gitea` as network.

Reply:
The most important thing is to highlight that both should be in the same network. And if not, that the communication should go via the docker0 gateway (usually `172.17.0.1`). Maybe it's worth pointing this out in more detail instead of just mentioning the hard-coded docket network value?